### DISCUSSION TITLE
Windows 7 Pro Useing question Windows7专业版使用时问题

### DISCUSSION THREAD

**Discussion by RenzheLee** [t1m0thyj/WinDynamicDesktop#373] [Q&A]
When softwear installed,it will pop up the fault box say: check C:\User\(Username)\Appdeta\Local\Programs\WinDynamicDesktop\log for details
Document contens::[2021-XX-XX 10:XX:32.391] System.IO.FileNotFoundException: Failed to load file or assembly "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" or one of its dependencies. The system can not find the file specified.
File name: "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"
   In WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   In WinDynamicDesktop.LocationIQService.HandleLocationSuccess(LocationIQData data, ScheduleDialog dialog)
   In WinDynamicDesktop.LocationIQService.GetLocationData(String locationStr, ScheduleDialog dialog)
   In WinDynamicDesktop.ScheduleDialog.<okButton_Click>d__11.MoveNext()
--- The end of the stack trace in the previous position where the exception was raised ---
   In System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)

Warning: Assembly binding logging is turned off.
To enable assembly binding failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There will be some performance penalty associated with assembly binding failure logging.
To turn off this feature, please remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

[2021-XX-XX XX:XX:43.208] System.IO.FileNotFoundException: Failed to load file or assembly "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" or one of its dependencies. The system can not find the file specified.
File name: "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"
   In WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   In WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   In WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   In WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   In System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   In System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   In System.Windows.Forms.ListView.WndProc(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   In System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Warning: Assembly binding logging is turned off.
To enable assembly binding failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There will be some performance penalty associated with assembly binding failure logging.
To turn off this feature, please remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

[2021-XX-XX XX:XX:16.943] System.IO.FileNotFoundException: Failed to load file or assembly "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" or one of its dependencies. The system can not find the file specified.
File name: "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"
   In WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   In WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   In WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   In WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   In System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   In System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   In System.Windows.Forms.ListView.WndProc(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   In System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Warning: Assembly binding logging is turned off.
To enable assembly binding failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There will be some performance penalty associated with assembly binding failure logging.
To turn off this feature, please remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

[2021-XX-XX XX:XX:23.103] System.IO.FileNotFoundException: Failed to load file or assembly "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" or one of its dependencies. The system can not find the file specified.
File name: "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"
   In WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   In WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   In WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   In WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   In System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   In System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   In System.Windows.Forms.ListView.WndProc(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   In System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   In System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Warning: Assembly binding logging is turned off.
To enable assembly binding failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There will be some performance penalty associated with assembly binding failure logging.
To turn off this feature, please remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].

[2021-XX-XX XX:XX:41.991] System.IO.FileNotFoundException: Failed to load file or assembly "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" or one of its dependencies. The system can not find the file specified.
File name: "netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"
   In WinDynamicDesktop.SunriseSunsetService.GetSolarData(DateTime date)
   In WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   In WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   In WinDynamicDesktop.ThemeDialog.listView1_S

How to solce this problem.HELP!!

**Comment by t1m0thyj** [comment]
Do you have .NET Framework 4.5 or newer installed? On Windows 7, you may have to install it before you can use WDD as documented here: https://github.com/t1m0thyj/WinDynamicDesktop#supported-devices

**Reply by RenzheLee** [reply]
Thank you.I'll try that

**Reply by RenzheLee** [reply]
Err... I see my .NET Framework Version is 4.6.1.So....... Is there any other way?

**Reply by t1m0thyj** [reply]
WDD 5.0 and later uses a newer version of .NET (6 instead of 4.5) and it's now bundled in the executable. This makes the app EXE much bigger but hopefully it solves your problem 🙂
